### ISSUE TITLE
Expressions: Hide the edit button for repeating group rows

### ISSUE THREAD

**Issue by olemartinorg** [Altinn/app-frontend-react#582]
### Description

In the expressions implementation (#540), it is now possible to hide the `addButton`, `saveButton` and `deleteButton` for a repeating group. It should also be possible to hide the edit button this way, and have the expression evaluate per-row so that each row can decide to have their edit button hidden on their own.

### Additional Information

- #540 
- #61

**Comment by StianVestli** [comment]
org/ssb

**Comment by olemartinorg** [comment]
@Magnusrm This should be easy now, after #61 and #64, right? :pray: Up for grabs if you want it!

**Comment by Magnusrm** [comment]
@olemartinorg I can definitely pick this up. 😄 
I imagine the expression part is already done, I guess this is mostly a matter of finding a good way to display different combos of edit and delete buttons in the rows. Ex: one row with only delete button and one with only edit button and one with no buttons. I imagine the header columns to be a bit tricky to handle with the current implementation, we might need to create a "action" column instead of "edit-button" and "delete-button" columns.

We might also need UX to decide how we want to display the different combos.

**Comment by olemartinorg** [comment]
@Magnusrm Great point! Tagging @Febakke and @mrosvik. I think we just implement this short-term, but possibly reconsider the design in the long term.
